modules/wallet/wallet_test.go

```diff
@@ -15,6 +16,16 @@ import (
 	"github.com/NebulousLabs/Sia/types"
 )
 
+var (
+	// Sometimes during testing, you get a statement
+	// `if err != expectedErr { return err }`. But if err == nil, then the
+	// calling function will think that the test has passed, even though the
+	// test failed. [This is a developer error, instead return
+	// errors.New("expecting..")] The new type of error catches when the
+	// developer has made a mistake and failed to catch a test suite failure.
+	errTestCompleted = errors.New("test complete")
```

Comment:
gonna veto this. Returning nil is a developer error, but so is forgetting to return `errTestCompleted` at the end of a test. In fact, I would say devs would be more prone to make the latter mistake, since it's unintuitive to return an "error" in the event of success.